### ISSUE TITLE
Lua error: @ytdl_hook.lua:59: bad argument #1 to 'len' (string expected, got nil)

### ISSUE THREAD

**Issue by rindeal** [mpv-player/mpv#4351]
### mpv version and platform

0.24.0 on GNU/Linux
yt-dl 2017.04.17

### Reproduction steps

```
$ mpv --msg-level=ytdl_hook=debug http://www.ceskatelevize.cz/ivysilani/10805121298-gejzir
```

### Expected behavior

The video starts playing

### Actual behavior

An error occurs.

### Log file

```
[ytdl_hook] Loading lua script @ytdl_hook.lua...
[ytdl_hook] loading mp.defaults
[ytdl_hook] loading @ytdl_hook.lua
Playing: http://www.ceskatelevize.cz/ivysilani/10805121298-gejzir/217562235000004/obsah/524193-drony
[ytdl_hook] Running: youtube-dl --no-warnings -J --flat-playlist --sub-format ass/srt/best --no-playlist --format bestvideo[ext=mp4]+bestaudio[ext=m4a]/best[ext=mp4]/best --all-subs -- http://www.ceskatelevize.cz/ivysilani/10805121298-gejzir/217562235000004/obsah/524193-drony
[ytdl_hook] youtube-dl succeeded!
[ytdl_hook] multi-arc video detected, building EDL
[ytdl_hook]
[ytdl_hook] stack traceback:
[ytdl_hook]     @ytdl_hook.lua:59: in function 'edl_escape'
[ytdl_hook]     @ytdl_hook.lua:119: in function 'edl_track_joined'
[ytdl_hook]     @ytdl_hook.lua:229: in function 'fn'
[ytdl_hook]     mp.defaults:518: in function 'handler'
[ytdl_hook]     mp.defaults:339: in function 'handler'
[ytdl_hook]     mp.defaults:458: in function 'call_event_handlers'
[ytdl_hook]     mp.defaults:495: in function 'dispatch_events'
[ytdl_hook]     mp.defaults:451: in function <mp.defaults:450>
[ytdl_hook]     [C]: at 0x00477ce0
[ytdl_hook]     [C]: at 0x00479180
[ytdl_hook] Lua error: @ytdl_hook.lua:59: bad argument #1 to 'len' (string expected, got nil)
[ytdl_hook] Exiting...
Cache fill:  0.06% (59766 bytes)
Failed to recognize file format.


Exiting... (Errors when loading file)
```

**Comment by wiiaboo** [comment]
Add `no-ytdl` to your mpv.conf and this version of ytdl_hook.lua to your scripts directory and check if if fixes that video and works with others of that site: https://github.com/wiiaboo/mpv/blob/fd4024ddeb430b3c3ea59960124ece71039cc327/player/lua/ytdl_hook.lua

**Comment by rindeal** [comment]
I used a simpler `mpv --no-ytdl --script=./ytdl_hook.lua ...` command, but yes, it works. Thank you.

**Comment by wiiaboo** [comment]
Fixed in https://github.com/mpv-player/mpv/commit/3526b95aafe11bb829c924a1f69877ec72406068

**Comment by tp0** [comment]
I assume this is fixed for 0.26?

I'm still stuck with 0.25 due to my Linux distrubution. As another workaround I downgraded youtube-dl to version 2017.07.30.1 (`pip install -U youtube-dl==2017.07.30.1`).

**Comment by wiiaboo** [comment]
This is not the same issue as https://github.com/mpv-player/mpv/issues/4728.
